.fixtures.yml

```diff
@@ -9,10 +9,22 @@ fixtures:
     classification:
       repo: "https://github.com/ploperations/ploperations-classification"
       ref: "a0ee04c65d89ca648113e96d15c5001c70fc718d"
+    ssh:
+      repo: "https://github.com/ploperations/ploperations-ssh"
+      branch: "windows"
```

Comment:
Bleh. I've put this at a branch so that it points in the right place until https://github.com/ploperations/ploperations-ssh/pull/10 is merged. At that point we should tag it and update this to point at that tag.